wordpress-mobile/WordPressCom-Stats-iOS
# Turns Insights Today's Stats into multiple rows on iPhone

Closes #290
Fixes #303, #292

Copies over some of logic and the cell used in the stats graph section for use in Insights Today's Stats. On the iPhone the layout is modified to use multiple rows rather than the 2x2 like previous or the 1x4 grid like on iPad.

iPad (original layout) - 
![ios simulator screen shot sep 1 2015 1 21 48 pm](https://cloud.githubusercontent.com/assets/373903/9615284/df3f218a-50bc-11e5-8f97-5b1620be2d76.png)

iPhone (new layout) -
![ios simulator screen shot sep 1 2015 1 24 39 pm](https://cloud.githubusercontent.com/assets/373903/9615295/e579b218-50bc-11e5-8873-24b30fc7c777.png)

Testing (using StatsDemo) - 
* View today's stats on iPhone, iPad and verify they are the same as the web
* Try rotation, refreshing, etc.

Needs Review: @jleandroperez, @jancavan
cc: @daniloercoli
